src/io/libxdrf.c

```diff
@@ -0,0 +1,1008 @@
+/*
```

Comment:
Should we really add this file???

Wouldn't it be better to use `find_package(libxdr)` to detect and external xdrfile library?